attest_fn_call/Makefile

```diff
@@ -29,21 +26,16 @@ tmp/x.wasm: $(SRCS) $(DEPS) | tmp check
 	@docker exec -u root $(CONTAINER_NAME) \
  		chown -R 1000:1000 /home/tinygo/go
 	@docker exec -w /src -u tinygo $(CONTAINER_NAME) \
- 		tinygo build $(TINYGO_FLAGS) -o tmp/x.wasm -target=wasi ./...
+ 		tinygo build $(TINYGO_FLAGS) -o main.wasm -target=wasi ./...
 	@docker stop $(CONTAINER_NAME) > /dev/null 2>&1 && docker rm $(CONTAINER_NAME) > /dev/null 2>&1 &
 
-.PHONY: build
-build: tmp/x.wasm
+out.json: main.wasm fn-call.json | check
+	@cat fn-call.json | bky-as attest-fn-call >out.json
 
 .PHONY: run
-run: check build
-	@echo "Running WASM module..."
-	@cat fn-call.json | bky-as attest-fn-call >tmp/out.json
-	@echo "Log:"
-	@jq -r '.transitive_attested_function_call.logs | @base64d' tmp/out.json
-	@echo "Output:"
-	@jq -r '.transitive_attested_function_call.claims.output | @base64d ' tmp/out.json
+run: out.json
```

Comment:
I'm keeping the `run` target for convenience of running everything, though it doesn't show up in the documentation, or in the tests.